samples/matter/common/src/zap_samples.yml

```diff
@@ -0,0 +1,54 @@
+# This file is used to generate the ZAP files for the samples.
+# Use this file as an argument to the west zap-generate command:
+#
+# west zap-generate -y zap_samples.yml
+#
+
+# Base dir related to ZEPHYR_BASE
+- base_dir: ../nrf
+
+# applications
+- name: matter_bridge
+  zap_file: applications/matter_bridge/src/default_zap/bridge.zap
```

Comment:
You missed zap file for bridge with on_off plug

Reply:
Right, done.